src/tile/tile.scss

```diff
@@ -63,27 +63,22 @@
       }
     }
 
-    > svg {
+    .iui-placeholder-icon {
```

Comment:
Ok, but I am not sure about the class name.. Why its placeholder? 
According to the code, if they pass there svg or any jsx element, this would be applied.

Reply:
Maybe tile-icon or thumbnail-icon?